main.c

```diff
@@ -94,6 +105,12 @@ static bool parse_args(int argc, char **args)
                 opt_trace = true;
                 continue;
             }
+#ifdef ENABLE_GDBRSP
+            if (!strcmp(arg, "--debug")) {
```

Comment:
How about changing the option name to `--gdbstub`?

See also: https://github.com/bochs-emu/Bochs/blob/master/bochs/config.cc